rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4280,10 +4281,9 @@ void opcode_018b(const ScriptArguments& args, const ScriptBlip blip, const Scrip
     @arg sound 
 */
 void opcode_018c(const ScriptArguments& args, ScriptVec3 coord, const ScriptSoundType sound) {
-    RW_UNIMPLEMENTED_OPCODE(0x018c);
-    RW_UNUSED(coord);
-    RW_UNUSED(sound);
-    RW_UNUSED(args);
+    auto world = args.getWorld();
+    auto name = world->sound.createSfxInstance(convertScriptIndexIntoSfx(sound));
+    world->sound.playSfx(name, coord, false, getScriptRange(sound));
```

Comment:
You call `convertScriptIndexIntoSfx` and `getScriptRange` here, where they both do a linear search to find the relevant SFX.

It would be better to have a single function like `SoundInstanceData* findScriptSoundSFXData(size_t sound)` so you can avoid doing the search twice, and more explicitly handle the invalid sound index case.